iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/iot/IoTConsensusServerImpl.java

```diff
@@ -844,8 +844,7 @@ public long getSearchIndex() {
   }
 
   public long getSyncLag() {
-    long minSyncIndex = getMinSyncIndex();
-    return getSearchIndex() - minSyncIndex;
+    return getSearchIndex() - getMinFlushedSyncIndex();
```

Comment:
It is recommended to use a local variable as before. Because the two get operation is not an atomic operation, subtracting the two function may result in a negative number.

Reply:
- getMinFlushedSyncIndex() + getSearchIndex() looks better :P